### PR TITLE
Compact layout

### PR DESCRIPTION
**Pull request by shinyichen**
* Make display of results and abstract text more compact.
* Subpage parsing fix

Screen Shot 2021-11-03 at 1 22 36 PM

Screen Shot 2021-11-03 at 1 22 57 PM

### CODE REVIEW

File: src/components/ResultList/Item/Item.tsx [hunk 1]
```diff
@@ -84,30 +84,29 @@ export const Item = (props: IItemProps): ReactElement => {
...
+            <div className="text-sm">
               {author.slice(0, 10).join('; ')}
               {author_count > 10 && <span className="text-xs italic"> and {author_count - 10} more</span>}
             </div>
           )}
           <div className="flex py-1">
             <span className="text-xs">
               {formattedPubDate}
-              {formattedPubDate && formattedBibstem ? ' | ' : ''}
+              {formattedPubDate && formattedBibstem ? <>&nbsp;&nbsp;·&nbsp;&nbsp;</> : ''}
```
Comment (thostetler) [inline src/components/ResultList/Item/Item.tsx:105]: maybe dumb question by is this better than doing:
```suggestion
              {formattedPubDate && formattedBibstem ? <span className="px-1">·</span> : ''}
```
or whatever padding would look good

File: src/components/AbstractSideNav/AbstractSideNav.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export interface IAbstractSideNavProps extends HTMLAttributes<HTMLDivElement> {
 
 export const AbstractSideNav = ({ doc }: IAbstractSideNavProps): ReactElement => {
   const router = useRouter();
-  const subPage = last(router.asPath.split('/'));
+  const subPage = last(router.asPath.split('/')).split('?')[0];
```
Comment (thostetler) [inline src/components/AbstractSideNav/AbstractSideNav.tsx:22]: There is a utility for splitting on `?`
```suggestion
  const basePath = useBaseRouterPath();
  const subPage = last(basePath);
```

**Review by thostetler — CHANGES_REQUESTED** [review]
LGTM - couple small changes

**Review by marblestation — APPROVED** [review]
I am not checking the implementation, only the look & feel, and I like it!

File: src/components/ResultList/Item/Item.tsx (hunk 1 above)
Comment (shinyichen) [reply src/components/ResultList/Item/Item.tsx:105]: You are right!

Comment (shinyichen) [reply src/components/ResultList/Item/Item.tsx:105]: Changed

**Pull request merged**